logging/fb/fluent-bit_config.configmap_open.yaml

```diff
@@ -195,7 +195,7 @@ data:
        Alias events_renames
        Name  modify
        Match kube.*
-       Condition Key_value_equals  PROPS__appname eventrouter
+       Condition Key_value_matches  PROPS__appname eventrouter*
```

Comment:
Is this change necessary and/or does it it improve things in some way?  If we do not change the Kubernetes label (see previous comment), do we need this change?

Reply:
I went ahead and reverted this change.  It should show up in the next update.